docs/build-the-docs.md

```diff
@@ -74,7 +88,7 @@ DOCSEARCH_ENABLED=true DOCSEARCH_ENGINE=lunr antora --pull \
 
 - You can add the `--clean` option to clean the build directory of any leftover artifacts from the previous build, including PDF's.
 
-### Update The Generated Search Index
+#### Update The Generated Search Index
```

Comment:
imho we have to do something better / clarifying here as it is very confusing when you look downwards to `update-the-search-index.md`

Reply:
I mean a better explanation why a search index needs updating

Reply:
This is fixed in #500. So long as it's merged after, then this will be resolved.